runtime/model-protobuf/src/test/java/io/aklivity/zilla/runtime/model/protobuf/internal/config/ProtobufModelConfigAdapterTest.java

```diff
@@ -201,8 +198,37 @@ public void shouldReadProtobufWithProto2Syntax()
 
         // THEN
         assertThat(converter.model, equalTo("protobuf"));
-        assertThat(converter.syntax, equalTo("proto2"));
         assertThat(converter.cataloged, hasSize(1));
         assertThat(converter.cataloged.get(0).name, equalTo("test0"));
     }
+    
+    @Test
+    public void shouldIgnoreLegacySyntaxFieldIfPresent()
+    {
+        // GIVEN - JSON with legacy syntax field (for backward compatibility)
+        String json =
+            "{" +
+                "\"model\":\"protobuf\"," +
+                "\"syntax\":\"proto2\"," +  // Legacy field, should be ignored
```

Comment:
This level of compatibility is not required as we haven't merged or released this yet.
Please remove concept of syntax from the schema.